src/schema/v2/artist/targetSupply/index.ts

```diff
@@ -10,25 +10,23 @@ import {
   GraphQLString,
   GraphQLInt,
 } from "graphql"
-import { ArtworkType, artworkConnection } from "schema/v2/artwork"
-import { pageable } from "relay-cursor-paging"
-import { convertConnectionArgsToGravityArgs } from "lib/helpers"
-import { createPageCursors } from "schema/v2/fields/pagination"
-import { connectionFromArray } from "graphql-relay"
+import { ArtworkType } from "schema/v2/artwork"
 import { deprecate } from "lib/deprecation"
+import { getRecentlySoldArtworksConnection } from "schema/v2/types/recentlySoldArtworksConnection"
+import { TargetSupplyMicrofunnelMetadata } from "schema/v2/types/targetSupplyMicrofunnelMetadata"
 
 const ArtistTargetSupplyType = new GraphQLObjectType<any, ResolverContext>({
   name: "ArtistTargetSupply",
   fields: {
     isTargetSupply: {
       description: "True if artist is in target supply list.",
       type: GraphQLBoolean,
-      resolve: (artist) => artist.target_supply,
+      resolve: artist => artist.target_supply,
```

Comment:
I'm not sure I understand the back and forth prettier churn here. As far as I'm aware, VS Code's prettier extension uses the version installed in node_modules; and I don't think I have any settings set anywhere to favor no parens around single function arguments.

Reply:
Man, yeah, this is driving me totally nuts in my code lately... I think its something to do with v2 + VSCode + some kind of mismatching. Will need to fiigure out whats up at some point :/